.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: php
 php:
-  - 5.2
```

Comment:
Everything in this file, apart from this change here, should be enough to fix it?